App/test/app/UserProfileTest.java

```diff
@@ -235,8 +235,8 @@ public void CancelActionPerformedTest (){
                 model = (DefaultTableModel) Bookings.getModel();
                 model.setRowCount(0);
                 int rowIndex = Bookings.getSelectedRow();
-                //int rowIndex = Bookings.setRowSelectionInterval(0, 0);
-                int bookid= (int) model.getValueAt(rowIndex, 0);
+                Bookings.setRowSelectionInterval(0, 0);
+                int bookid= (int) model.getValueAt(Bookings, 0);
```

Comment:
Arwa this bit is almost perfect,

Just it you need to set it then get it.

Then, you can get the result to rowIndex.

Reply:
`
Bookings.setRowSelectionInterval(0, 0);
int rowIndex = Bookings.getSelectedRow();
int bookid= (int) model.getValueAt(rowIndex, 0);
`
It looks more like this so you set it get it the get the booking id